Formula/dum.rb

```diff
@@ -0,0 +1,25 @@
+class Dum < Formula
+  desc "Npm scripts runner written in Rust"
+  homepage "https://github.com/egoist/dum"
+  url "https://github.com/egoist/dum/archive/v0.1.8.tar.gz"
+  sha256 "0c8337d79e0e331b2dbd14e615adc9d359d3491632e21d479a306c9a2845009c"
+  license "MIT"
+  head "https://github.com/egoist/dum.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
Is this needed? i.e. is the default `livecheck` wrong if you omit this?

Reply:
I just copy from `fnm.rb`. The default behavior is detecting Github release?

Reply:
Try deleting it and then check the output of `brew livecheck dum`. If it's wrong we can keep this. If it's not then we don't need it. Whether you need it or not depends on the repository.

Reply:
Yes, provided the url is a github repo

Reply:
It works!